src/main/java/com/aws/greengrass/mqtt/bridge/BridgeConfig.java

```diff
@@ -103,11 +103,17 @@ private static URI getBrokerUri(Topics configurationTopics) throws InvalidConfig
                 configurationTopics.findOrDefault(DEFAULT_BROKER_URI, KEY_BROKER_SERVER_URI));
         String brokerUri = Coerce.toString(
                 configurationTopics.findOrDefault(brokerServerUri, KEY_BROKER_URI));
+        URI uri;
         try {
-            return new URI(brokerUri);
+            uri = new URI(brokerUri);
         } catch (URISyntaxException e) {
             throw new InvalidConfigurationException("Malformed " + KEY_BROKER_URI + ": " + brokerUri, e);
         }
+
+        if (uri.getPort() < 0) {
```

Comment:
We should have a default shouldn't we? Is this really what we did before your change?

Reply:
updated PR description, just checked paho docs,  if no port provided it'll use 1883/8883 as sensible, so not an existing bug

Reply:
will change to default to 1883/8883 based on the protocol

Reply:
so throwing an exception on missing port would break upgrade path.... will make sure not to do that